src/components/modals/authentification/ModalAuthInitView.tsx

```diff
@@ -0,0 +1,123 @@
+
+/**
+ * Modal vue de base avec boutons google, facebook, apple non actifs
+ */
+type Props = {
+  setContenuModal: React.Dispatch<React.SetStateAction<"init" | "connexion" | "inscription" | 'password-forget'>>
+};
+
+const ModalAuthInitView = ({ setContenuModal}: Props ) => {
+
+  return (
+    <>
+      <div className="flex flex-col items-center w-full px-4 pb-6">
+        <h1 className="text-2xl text-center">Bienvenue !</h1>
+
+        <button
+          type="button"
+          role="button"
+          className="h-11 w-full flex justify-center items-center border border-solid border-vintedTextGrisClair rounded mt-6 cursor-not-allowed"
+          disabled
+        >
+          <span className="pr-[6px]">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 48 48"
+              width="24px"
+              height="24px"
+            >
+              <path
+                fill="#039be5"
+                d="M24 5A19 19 0 1 0 24 43A19 19 0 1 0 24 5Z"
+              />
+              <path
+                fill="#fff"
+                d="M26.572,29.036h4.917l0.772-4.995h-5.69v-2.73c0-2.075,0.678-3.915,2.619-3.915h3.119v-4.359c-0.548-0.074-1.707-0.236-3.897-0.236c-4.573,0-7.254,2.415-7.254,7.917v3.323h-4.701v4.995h4.701v13.729C22.089,42.905,23.032,43,24,43c0.875,0,1.729-0.08,2.572-0.194V29.036z"
+              />
+            </svg>
+          </span>
+          <span className="text-vintedTextGrisClair">
+            Continuer avec Facebook
+          </span>
+        </button>
+
+        <button
+          type="button"
+          role="button"
```

Comment:
Amélioration repérée avec Sonarlint : The element button has an implicit role of button. Defining this explicitly is redundant and should be avoided.sonarlint(typescript:S6822)

Reply:
corrigé